tests/Sulu/Component/Security/Authorization/SecurityContextVoterTest.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Security\Authorization;
+
+use Prophecy\Argument;
+use Prophecy\PhpUnit\ProphecyTestCase;
+use Sulu\Bundle\SecurityBundle\Entity\Group;
+use Sulu\Bundle\SecurityBundle\Entity\Permission;
+use Sulu\Bundle\SecurityBundle\Entity\Role;
+use Sulu\Bundle\SecurityBundle\Entity\User;
+use Sulu\Bundle\SecurityBundle\Entity\UserGroup;
+use Sulu\Bundle\SecurityBundle\Entity\UserRole;
+use Symfony\Component\Security\Acl\Exception\AclNotFoundException;
+use Symfony\Component\Security\Acl\Model\AclProviderInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+
+class SecurityContextVoterTest extends ProphecyTestCase
+{
+    protected $permissions = array(
+        'view' => 64,
+        'add' => 32,
+        'edit' => 16,
+        'delete' => 8,
+        'archive' => 4,
+        'live' => 2,
+        'security' => 1
```

Comment:
Not related, but making the values of the these configurable is dangerous. They should at least be validated somewhere.